jgt_tools/utils.py

```diff
@@ -24,7 +24,8 @@ def execute_command_list(commands_to_run, verbose=True):
 
 
 def _load_defaults():
-    with open("data/defaults.csv") as f:
+    defaults = Path(__file__).parent / "data" / "defaults.csv"
```

Comment:
I generally prefer this pathlib style, can we make this a module level value for greater visibility and it's own doc string?